ammonia_test_script.py

```diff
@@ -0,0 +1,40 @@
+#####################################################################
+# This script tests the ammonia fitter                              #
+# with fake data produced by ammonia_model.n_modelfunc              #
+# using the xarr from G031.947+00.076_nh3_11_Tastar.fits [for now]  #
+#####################################################################
+
+from pyspeckit.spectrum.models import ammonia
+from pyspeckit import Spectrum
+from pyspeckit.spectrum.units import SpectroscopicAxis
+from astropy import units as u
+import numpy as np
+import matplotlib.pyplot as plt
+
+parameter_error_amplitude = [5., 5., 5., 5., 5., 5.]
+params = [2.3,2,0.,0.84,0.2,0.43]
+
+sp = Spectrum('G031.947+00.076_nh3_11_Tastar.fits')
+xarr = SpectroscopicAxis(np.linspace(-250, 385, 8192), refX=23694500000.0*u.Hz)
```

Comment:
Ahh, I see.  Yes, probably the old defaults are wrong and I was reacting to my old mistake.  The default should be `None` or `u.dimensionless_unscaled`. 

Also, let's add a warning:
```
if unit is None and not hasattr(data,'unit'):
    log.warn("Units were left as 'None'.  If this is intentional, you can disable this warning by passing unit=u.dimensionless_unscaled")
```